blog/_posts/2020-10-04-vsm-and-kaizen-modules-added.md

```diff
@@ -0,0 +1,26 @@
+Following the positive community reaction to the launch of the Online DevOps Dojo, we are pleased to announce we have added two more modules to the dojo
```

Comment:
Should end in either a period or maybe a colon to introduce bullets if that's the intent